safe_locking_service/campaigns/tests/test_views.py

```diff
@@ -176,9 +178,17 @@ def test_retrieve_campaign_view(self):
 class TestActivitiesUploadView(TestCase):
     def setUp(self):
         self.client = Client()
-        self.admin_user = User.objects.create_superuser(
-            "admin", "admin@example.com", "adminpass"
-        )
+        # Create a normal user
+        self.user = User.objects.create_user("admin", "admin@example.com", "adminpass")
```

Comment:
Just to follow the idea of normal user, could you change admin by user?
```suggestion
        self.user = User.objects.create_user("user", "user@example.com", "userpass")
```